partner_address_two_lines/models/res_partner.py

```diff
@@ -1,16 +1,51 @@
 # Copyright 2020 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from odoo import models
+from odoo import api, models
 
 
 class ResPartner(models.Model):
     _inherit = "res.partner"
 
-    def _get_contact_name(self, partner, name):
+    def _get_complete_name(self):
+        res = super()._get_complete_name()
         if self.env.context.get("_two_lines_partner_address"):
```

Comment:
Print the Quotation report (of sale.order), you can the `Gemini Furniture` and `YourCompany` were shown twice.
I checked v16 and saw that you should update this line to be:` if (self.company_name or self.parent_id) and not self.is_company and self.env.context.get("_two_lines_partner_address"):`
![image](https://github.com/trisdoan/partner-contact/assets/8724793/4784124e-ebe5-48de-9f5c-c466b14658bd)